### PR TITLE
Add support for audio to recordings UI & removed legacy audio UI

### PR DESCRIPTION
**Pull request by mjs**
- recording type now shown on query results screen
- limit field now merged onto same line as Search button
- added support for audio on the view recording page
- show more metadata on the view page
- removed legacy audio UI

### CODE REVIEW

**Comment by mjs** [comment]
image

**Comment by mjs** [comment]
Example of an audio file. Layout is a bit messy but we can make it better in the Vue version.

image

File: public/javascripts/viewRecording.js [hunk 1]
```diff
@@ -115,20 +115,63 @@ function getRecordingSuccess(result) {
...
+  document.getElementById('relative-to-dawn').innerHTML = recording.relativeToDawn;
+  document.getElementById('location').innerHTML = formatLocation(recording.location);
+  document.getElementById('battery-level').innerHTML = recording.batteryLevel;
+  document.getElementById('battery-charging').innerHTML = recording.batteryCharging;
+  document.getElementById('airplane-mode').innerHTML = recording.airplaneModeOn;
+  document.getElementById('version').innerHTML = recording.version;
+  document.getElementById('additional-metadata').innerHTML = formatMetadata(recording.additionalMetadata);
+  document.getElementById('comment-text').value = recording.comment;
+
+  if (result.downloadFileJWT) {
```
Comment (CameronRP) [inline public/javascripts/viewRecording.js:139]: Currently you will get a JWT for downloading the file even if it hasn't been processed. Just that the key in the JWT will be null. This should probably be changed on the API side though.

**Review by CameronRP — APPROVED** [review]
Looks good. Just one thing to note.

**Review by arthurmcgregor — APPROVED** [review]
Looks good to me :-) Would you want to have audio/video/both as a filter option? Or would device selection be enough of a filter?

I've added an issue on the Vuex repo detailing the changes we'll need to make over there to match what you've done here...

**Comment by mjs** [comment]
@arthurmcgregor I was thinking about a audio/video/both filter option too. I'll do this as a follow up PR.

Thanks for recording what needs to be done for the Vue repo. I'm hoping to tackle a few smaller tasks like this for the Vue port so I can get more familiar with that codebase.

File: public/javascripts/viewRecording.js (hunk 1 above)
Comment (mjs) [reply public/javascripts/viewRecording.js:139]: Definitely should be fixed in the API server. https://github.com/TheCacophonyProject/cacophony-api/issues/130

**Pull request merged**